### PR TITLE
Fix InitializationTest

### PR DESCRIPTION
**Pull request by drcrallen**

### CODE REVIEW

**Comment by guobingkun** [comment]
:+1:  
It passed on Travis because ```extension``` directory doesn't exist on travis machine, it failed on local machine because ```extension``` directory exists, probably due to a previous ```mvn install```.

**Comment by gianm** [comment]
:+1:

File: server/src/test/java/io/druid/initialization/InitializationTest.java [hunk 1]
```diff
@@ -187,12 +187,20 @@ public void testGetLoadedModules()
   }
 
   @Test
-  public void testGetExtensionFilesToLoad_non_exist_extensions_dir()
+  public void testGetExtensionFilesToLoad_non_exist_extensions_dir() throws IOException
   {
+    final File tmpDir = temporaryFolder.newFolder();
+    Assert.assertTrue("could not create missing folder", !tmpDir.exists() || tmpDir.delete());
     Assert.assertArrayEquals(
         "Non-exist root extensionsDir should return emply array of File",
```
Comment (rasahner) [inline server/src/test/java/io/druid/initialization/InitializationTest.java]: emply?  empty?

Comment (guobingkun) [inline server/src/test/java/io/druid/initialization/InitializationTest.java]: Thanks for the catch! Sent https://github.com/druid-io/druid/pull/2068.

**Pull request merged**